package.json

```diff
@@ -30,17 +30,26 @@
     "@hapi/eslint-config-hapi": "^13.0.0",
     "@hapi/eslint-plugin-hapi": "^4.3.3",
     "@hapi/hapi": "^20.2.1",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
     "eslint": "^7.7.0",
     "nyc": "^15.1.0",
     "proxyquire": "^2.1.3",
-    "tape": "^5.0.1"
+    "tape": "^5.0.1",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "commitizen": "^4.2.4",
+    "cz-conventional-changelog": "^3.3.0",
+    "semantic-release": "^18.0.1"
```

Comment:
These should be sorted alphabetically. If not, any npm CLI command will reorder them and create a modified file that might conflict with committing (during the workflow).